Openshift-EE/pipelines/ElasticSearch/stages/4-chaos/IOWW-cstor-pool-delete/cstor-pool-delete

```diff
@@ -1,5 +1,101 @@
 #!/bin/bash
 set -x
 
-echo "**cstor pool delete***"
-sleep 2
+pod() {
+sshpass -p $pass ssh -o StrictHostKeyChecking=no $user@$ip -p $port 'cd e2e-openshift && bash Openshift-EE/pipelines/ElasticSearch/stages/4-chaos/IOWW-cstor-pool-delete/cstor-pool-delete node '"'$CI_JOB_ID'"'' '"'$CI_PIPELINE_ID'"' '"'$CI_COMMIT_SHA'"'
+}
+
+node() {
+
+source ~/.profile
+#github token to push the test result into github mayadata-io/e2e-openshift repository.
+#This token is set as an env in ~/.profile in the test cluster.
+gittoken=$(echo "$github_token")
+job_id=$(echo $1) #Gitlab job id obtain from the gitlab env ($CI_JOB_ID).
+pipeline_id=$(echo $2) # Gitlab pipeline id obtained from gitlab env ($CI_PIPELINE_ID).
+commit_id=$(echo $3) #Gitlab commit id Obtained fron gilab env ($CI_COMMIT_SHA).
+case_id=IOWW #test case ID.
+
+time="date"
+current_time=$(eval $time)
+
+present_dir=$(pwd)
+echo $present_dir
+#Creating e2e custom resource result for the test.
+bash Openshift-EE/utils/e2e-cr jobname:pool-delete jobphase:Waiting 
+bash Openshift-EE/utils/e2e-cr jobname:pool-delete jobphase:Running init_time:"$current_time" jobid:"$job_id" pipelineid:"$pipeline_id" testcaseid:"$case_id"
+bash Openshift-EE/utils/e2e-cr jobname:pool-kill jobphase:Waiting      #cstor-pool-kill job should not be executed till pool-delete is completed
+
```

Comment:
@sushma1118 create cr status as waiting for `jobname:app-pod-failure jobphase:Waiting`

Reply:
ok